### ISSUE TITLE
Failing to load interactive widget

### ISSUE THREAD

**Issue by dav92lee** [menpo/menpowidgets#24]
Hello friends!  I am struggling to get this interactive widget to show up in my jupyter notebook.  Is there some flag I should set?  Would love any tips.

![image](https://cloud.githubusercontent.com/assets/6377660/24447084/6bc07aa6-148d-11e7-98b5-2d7209b58e7f.png)

![image](https://cloud.githubusercontent.com/assets/6377660/24447106/84972688-148d-11e7-8ab4-b4125021f041.png)

Best!

**Comment by jabooth** [comment]
Hmm that's weird!
Did you install with conda? If so can you provide the output of the following in your conda environment:
```
conda list
```

**Comment by dav92lee** [comment]
yup!  installed with conda:
```

# packages in environment at /Users/swipe/miniconda2:
#
appnope                   0.1.0                    py27_0  
appnope                   0.1.0                     <pip>
apptools                  4.4.0                    py27_0    menpo
backports                 1.0                      py27_0  
backports-abc             0.5                       <pip>
backports.shutil-get-terminal-size 1.0.0                     <pip>
backports_abc             0.5                      py27_0  
bleach                    1.5.0                    py27_0  
bleach                    2.0.0                     <pip>
boost                     1.59.0                   py27_0    menpo
bzip2                     1.0.6                         3  
certifi                   2017.1.23                 <pip>
cffi                      1.9.1                    py27_0  
conda                     4.3.14                   py27_0  
conda-env                 2.6.0                         0  
configobj                 5.0.6                    py27_0  
configparser              3.5.0                    py27_0  
configparser              3.5.0                     <pip>
cryptography              1.7.1                    py27_0  
cycler                    0.10.0                   py27_0  
cyffld2                   0.2.4                    py27_0    menpo
cypico                    0.2.7                    py27_0    menpo
cyrasterize               0.3.2                    py27_0    menpo
cyvlfeat                  0.4.5                    py27_0    menpo
decorator                 4.0.11                    <pip>
decorator                 4.0.11                   py27_0  
dlib                      18.18                    py27_2    menpo
docopt                    0.6.2                    py27_0  
entrypoints               0.2.2                     <pip>
entrypoints               0.2.2                    py27_1  
enum34                    1.1.6                    py27_0  
envisage                  4.5.1                    py27_0    menpo
ffmpeg                    2.8.6                         0    menpo
fftw                      3.3.4                         0    menpo
freetype                  2.5.5                         2  
funcsigs                  1.0.2                    py27_0  
functools32               3.2.3.post2               <pip>
functools32               3.2.3.2                  py27_0  
get_terminal_size         1.0.0                    py27_0  
glew                      2.0.0                         0    menpo
glfw3                     3.2.1                         0    menpo
hdf5                      1.8.17                        1  
html5lib                  0.999                    py27_0  
html5lib                  0.999999999               <pip>
icu                       54.1                          0  
idna                      2.2                      py27_0  
ipaddress                 1.0.18                   py27_0  
ipykernel                 4.5.2                     <pip>
ipykernel                 4.5.2                    py27_0  
ipython                   4.2.0                    py27_1  
ipython                   5.3.0                     <pip>
ipython-genutils          0.2.0                     <pip>
ipython_genutils          0.2.0                    py27_0  
ipywidgets                6.0.0                     <pip>
ipywidgets                4.1.1                    py27_0  
jbig                      2.1                           0  
jinja2                    2.9.5                    py27_0  
Jinja2                    2.9.5                     <pip>
jpeg                      9b                            0  
jsonschema                2.5.1                    py27_0  
jsonschema                2.6.0                     <pip>
jupyter                   1.0.0                     <pip>
jupyter                   1.0.0                    py27_3  
jupyter-client            5.0.0                     <pip>
jupyter-console           5.1.0                     <pip>
jupyter-core              4.3.0                     <pip>
jupyter_client            5.0.0                    py27_0  
jupyter_console           5.1.0                    py27_0  
jupyter_core              4.3.0                    py27_0  
libiconv                  1.14                          0  
libpng                    1.6.27                        0  
libtiff                   4.0.6                         3  
libxml2                   2.9.4                         0  
MarkupSafe                1.0                       <pip>
markupsafe                0.23                     py27_2  
matplotlib                1.5.3               np111py27_1  
mayavi                    4.5.0                    py27_0    menpo
menpo                     0.7.7                    py27_0    menpo
menpo3d                   0.4.2                    py27_0    menpo
menpocli                  0.0.2                    py27_0    menpo
menpodetect               0.4.0                    py27_0    menpo
menpofit                  0.4.1                    py27_0    menpo
menpoproject              2.0                        py_0    menpo
menpowidgets              0.2.1                    py27_0    menpo
mistune                   0.7.4                     <pip>
mistune                   0.7.4                    py27_0  
mkl                       2017.0.1                      0  
mock                      2.0.0                    py27_0  
nbconvert                 5.1.1                     <pip>
nbconvert                 5.1.1                    py27_0  
nbformat                  4.3.0                     <pip>
nbformat                  4.3.0                    py27_0  
nose                      1.3.7                    py27_1  
notebook                  4.4.1                    py27_0  
notebook                  4.4.1                     <pip>
numpy                     1.11.3                   py27_0  
olefile                   0.44                     py27_0  
opencv3                   3.1.0                    py27_0    menpo
openssl                   1.0.2k                        0  
pandas                    0.19.2              np111py27_1  
pandocfilters             1.4.1                    py27_0  
pandocfilters             1.4.1                     <pip>
path.py                   10.1                     py27_0  
pathlib                   1.0                      py27_1    menpo
pathlib2                  2.2.1                     <pip>
pathlib2                  2.2.0                    py27_0  
pbr                       1.10.0                   py27_0  
pexpect                   4.2.1                     <pip>
pexpect                   4.2.1                    py27_0  
pickleshare               0.7.4                     <pip>
pickleshare               0.7.4                    py27_0  
pillow                    4.0.0                    py27_1  
pip                       9.0.1                    py27_1  
prompt-toolkit            1.0.14                    <pip>
prompt_toolkit            1.0.13                   py27_0  
ptyprocess                0.5.1                    py27_0  
ptyprocess                0.5.1                     <pip>
pyasn1                    0.1.9                    py27_0  
pycosat                   0.6.1                    py27_1  
pycparser                 2.17                     py27_0  
pyface                    5.1.0                    py27_0    menpo
pygments                  2.2.0                    py27_0  
Pygments                  2.2.0                     <pip>
pyopenssl                 16.2.0                   py27_0  
pyparsing                 2.1.4                    py27_0  
pyqt                      5.6.0                    py27_2  
python                    2.7.13                        0  
python-dateutil           2.6.0                     <pip>
python-dateutil           2.6.0                    py27_0  
python.app                1.2                      py27_4  
pytz                      2016.10                  py27_0  
pyzmq                     16.0.2                   py27_0  
pyzmq                     16.0.2                    <pip>
qt                        5.6.2                         0  
qtconsole                 4.2.1                     <pip>
qtconsole                 4.2.1                    py27_1  
readline                  6.2                           2  
requests                  2.12.4                   py27_0  
ruamel_yaml               0.11.14                  py27_1  
scandir                   1.5                      py27_0  
scandir                   1.5                       <pip>
scikit-learn              0.18.1              np111py27_1  
scipy                     0.19.0              np111py27_0  
setuptools                27.2.0                   py27_0  
simplegeneric             0.8.1                     <pip>
simplegeneric             0.8.1                    py27_1  
singledispatch            3.4.0.3                   <pip>
singledispatch            3.4.0.3                  py27_0  
sip                       4.18                     py27_0  
six                       1.10.0                   py27_0  
sqlite                    3.13.0                        0  
ssl_match_hostname        3.4.0.2                  py27_1  
tbb                       4.3_20141023                  0    menpo
terminado                 0.6                      py27_0  
terminado                 0.6                       <pip>
testpath                  0.3                       <pip>
testpath                  0.3                      py27_0  
tk                        8.5.18                        0  
tornado                   4.4.2                     <pip>
tornado                   4.4.2                    py27_0  
traitlets                 4.3.2                    py27_0  
traitlets                 4.3.2                     <pip>
traits                    4.5.0                    py27_0    menpo
traitsui                  5.1.0                    py27_0    menpo
vlfeat                    0.9.20                        1    menpo
vtk                       7.0.0                    py27_0    menpo
wcwidth                   0.1.7                    py27_0  
wcwidth                   0.1.7                     <pip>
webencodings              0.5                       <pip>
wheel                     0.29.0                   py27_0  
widgetsnbextension        2.0.0                    py27_0  
widgetsnbextension        2.0.0                     <pip>
xz                        5.2.2                         1  
yaml                      0.1.6                         0  
zlib                      1.2.8                         3 
```

**Comment by nontas** [comment]
I think this is an issue with `ipywidgets` (https://github.com/jupyter/notebook/issues/1386). It seems that `ipywidgets=4.1.1` do not appear in `notebook>=4.2`...

They have fixed this issue for `ipywidgets>=5.0` so this will be fixed by https://github.com/menpo/menpowidgets/pull/22. Maybe try to downgrade `notebook` and see if it is fixed until then?

**Comment by dav92lee** [comment]
Struggling with downgrading without version conflicts.  Would mean the world if one of you guys could put up your conda list where the interactive widget works?

specifically with `widgetsnbextension` and `notebook`

Best!

**Comment by HaoyangWang** [comment]
Same with ipywidgets 6.0 here...

update: solved by downgrading widgetsnbextension to 2.0
